Gemfile

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
-source 'https://rubygems.org'
+
+source "https://rubygems.org"
 
 # Specify your gem's dependencies in circlemator.gemspec
 gemspec
 
-gem 'rf-stylez', git: 'https://github.com/rainforestapp/rf-stylez.git', branch: 'upgrade_versions'
```

Comment:
I guess this means we always have to build and push the `rf-stylez` gem (and use the gem installer) when we want to update circlemator?

Reply:
I had previously added to pick rf-stylez from a branch because there was kind of a circular dependency where I needed to upgrade both to ruby 3.2.0 at the same time, or none would pass.

Now I removed the dependency from rf-stylez to circlemator (it just uses pronto directly), and have pushed a new rf-stylez version. So now I could remove this code to pick from a branch and it will pick from the latest deployed version.

But yes, in some cases if we want to upgrade circlemator, we might need to upgrade rf-stylez first. However, we have a new ticket with the purpose of deprecating circlemator, so I guess we should be fine.